undefined symbol `__sync_synchronize_dmb' referenced in expression

When trying to compile:
```plain
$ make
ipc.c
main.c
errf.c
srv.c
spinwait.s
start.s
svc.s
linking 3ds_pdn.elf
/opt/devkitpro/devkitARM/bin/../lib/gcc/arm-none-eabi/14.1.0/../../../../arm-none-eabi/bin/ld:--defsym:1: undefined symbol `__sync_synchronize_dmb' referenced in expression
collect2: error: ld returned 1 exit status
make[1]: *** [/opt/devkitpro/devkitARM/3ds_rules:44: /home/user/Documents/Programming/DS/Compiling/3ds_pdn/3ds_pdn.elf] Error 1
make: *** [Makefile:107: build] Error 2
```

--- a/source/sync.s
+++ b/source/sync.s
@@ -1,0 +1,11 @@
+.syntax unified
+.arch armv6k
+.arm
+.global __sync_synchronize_dmb
+.global __dmb
+
+__dmb:
+__sync_synchronize_dmb:
+	mov r0, #0
+	mcr p15, 0, r0, c7, c10, 5
+	bx lr

